src/PSR-12/CodeStyle.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * File doc
+ * PHP version 7.4
+ *
+ * @category Category
+ * @package  Package
+ * @author   Octavio <octavio.marchi@gmail.com>
+ * @license  http://www.opensource.org/licenses/mit-license.html  MIT License
+ * @link     5
+ */
+
+/**
+ * Class doc
+ *
+ * @category Category
+ * @package  Package
+ * @author   Octavio <octavio.marchi@gmail.com>
+ * @license  http://www.opensource.org/licenses/mit-license.html  MIT License
+ * @link     5
+ */
+class CodeStyle
```

Comment:
:no_entry_sign: **Error**: Each class must be in a namespace of at least one level (a top-level vendor name) (*PSR1.Classes.ClassDeclaration.MissingNamespace*).